src/modules/module_29600.c

```diff
@@ -152,13 +160,18 @@ int module_hash_decode (MAYBE_UNUSED const hashconfig_t *hashconfig, MAYBE_UNUSE
   if (tmp_len != 0x50) return (PARSER_HASH_LENGTH);
 
   
-  u32* whole_digest = (u32*)tmp_buf;
+  u32* whole_digest = (u32*) tmp_buf;
 
   // Penultimate block, i.e. IV, xored with a whole padding block
-  terra->ct_block_a[0] = byte_swap_32(whole_digest[0xc] ^ 0x10101010);
-  terra->ct_block_a[1] = byte_swap_32(whole_digest[0xd] ^ 0x10101010);
-  terra->ct_block_a[2] = byte_swap_32(whole_digest[0xe] ^ 0x10101010);
-  terra->ct_block_a[3] = byte_swap_32(whole_digest[0xf] ^ 0x10101010);
+  terra->ct[0] = byte_swap_32(whole_digest[0xc] ^ 0x10101010);
+  terra->ct[1] = byte_swap_32(whole_digest[0xd] ^ 0x10101010);
+  terra->ct[2] = byte_swap_32(whole_digest[0xe] ^ 0x10101010);
+  terra->ct[3] = byte_swap_32(whole_digest[0xf] ^ 0x10101010);
```

Comment:
Referring to style recommendations there should be space between function name and bracket.

```suggestion
  terra->ct[0] = byte_swap_32 (whole_digest[0xc] ^ 0x10101010);
  terra->ct[1] = byte_swap_32 (whole_digest[0xd] ^ 0x10101010);
  terra->ct[2] = byte_swap_32 (whole_digest[0xe] ^ 0x10101010);
  terra->ct[3] = byte_swap_32 (whole_digest[0xf] ^ 0x10101010);
```

Reply:
Correct